Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_wfsi.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/IPC_HLE/WII_IPC_HLE_Device_wfsi.h"
+
+#include <mbedtls/aes.h>
```

Comment:
Oh, I wasn't aware that we wanted to avoid indirect includes even for a single .cpp/.h pair. Never mind then.